SwiftGit2Tests/Fixtures/Fixtures.swift

```diff
@@ -47,12 +53,20 @@ final class Fixtures {
 	}
 	
 	func unzipFileAtURL(fileURL: NSURL, intoDirectoryAtURL directoryURL: NSURL) {
+#if os(OSX)
+
 		let task = NSTask()
 		task.launchPath = "/usr/bin/unzip"
 		task.arguments = [ "-qq", "-d", directoryURL.path!, fileURL.path! ]
-		
+            
 		task.launch()
 		task.waitUntilExit()
+
+#else
+
+		assertionFailure("Tests not supported on iOS yet")
```

Comment:
I haven't had much time to research libraries for unzipping on iOS. I think it would be best to handle that in another PR. I'll revise the description to make it clear that tests aren't included.